src/slidepane/SlidePane.ts

```diff
@@ -101,7 +103,7 @@ export default class SlidePane extends SlidePaneBase<SlidePaneProperties> {
 		} = this.properties;
 
 		// Current pointer position
-		const currentX = event.type === 'touchend' ? event.changedTouches[0].screenX : event.pageX;
+		const currentX = type === 'touchend' ? event.changedTouches[0].screenX : event.pageX;
```

Comment:
If you're going to grab `target` and `type`, might as well destructure fully and grab `changedTouches` too.